protocol/contact.go

```diff
@@ -58,7 +58,7 @@ type Contact struct {
 	SystemTags []string `json:"systemTags"`
 
 	DeviceInfo    []ContactDeviceInfo `json:"deviceInfo"`
-	TributeToTalk string              `json:"tributeToTalk,omitEmpty"`
+	TributeToTalk string              `json:"tributeToTalk,omitempty"`
```

Comment:
Potentially breaking change but tests pass. cc @cammellos

Reply:
This is ok, I believe, tributeToTalk is not used